service/raft/group0_state_machine.cc

```diff
@@ -348,7 +348,7 @@ future<> group0_state_machine::transfer_snapshot(raft::server_id from_id, raft::
 
     co_await _sp.mutate_locally({std::move(history_mut)}, nullptr);
   } catch (const abort_requested_exception&) {
-    throw raft::request_aborted();
+      throw raft::request_aborted(format("Abort requested while transferring snapshot from ID/IP: {}/{}, snapshot descriptor id: {}",  snp.idx, from_ip, snp.id));
```

Comment:
The ID/IP should refer to the source host from which we are transferring the snapshot. But you passed `snp.idx` as the ID which doesn't make sense. It should be `from_id`.

FWIW the snapshot index (`snp.idx`) is also useful info, so you could include it at the end additionally to the snapshot id.